cocos/rendering/geometry-renderer.ts

```diff
@@ -1017,10 +1017,16 @@ export class GeometryRenderer {
         }
     }
 
-    public addSpline (spline: geometry.Spline, color: Color, index = 0xffffffff, knotSize = 0.5, segments = 32, depthTest = true): void {
+    public addSpline (spline: geometry.Spline, color: Color, index = 0xffffffff, knotSize = 0.5, segments = 32, depthTest = true, useTransform = false, transform = new Mat4()): void {
```

Comment:
Will it create a Mat4 instance if it is not passed?

Reply:
It is better use Mat4.IDENTITY.

Reply:
Or maybe can just change the API to
```typescript
 public addSpline (spline: geometry.Spline, color: Color, index = 0xffffffff, knotSize = 0.5, segments = 32, depthTest = true, transform?: Mat4)
```

Reply:
Just because to keep the same API style in this class. You could refer to the other member functions.
The old style was using `default parameter`, I don't want to change that since it will not be unified with other member functions.

Reply:
> Will it create a Mat4 instance if it is not passed?

Yep, it will create the Mat4 instance here if transform is not passed in. But the reason is that I don't want make the API inconsistent.

Reply:
> It is better use Mat4.IDENTITY.

I agree this.